lambda/app/src/controllers/team.ts

```diff
@@ -185,12 +186,9 @@ export const updateMemberInTeam = async (teamId: number, userId: number, data: {
 
 export const requestServiceAccount = async (session: Session, userId: number, teamId: number, requester: string) => {
   const teamIdLiteral = getTeamIdLiteralOutOfRange(userId, teamId, ['admin']);
-  const integrations = await listIntegrationsForTeam(session, teamId);
+  const integrations = await listIntegrationsForTeam(session, teamId, 'gold');
```

Comment:
is there a reason this only lists for gold and not silver?

Reply:
Only those teams that have gold service type integrations are allowed to use the service account